go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	code.cloudfoundry.org/garden v0.0.0-20210208153517-580cadd489d2
 	code.cloudfoundry.org/lager v2.0.0+incompatible
 	github.com/CycloneDX/cyclonedx-go v0.7.1
-	github.com/DataDog/agent-payload/v5 v5.0.90-0.20230717070528-bfb5d051f2ab
```

Comment:
So this was introduced here: https://github.com/DataDog/datadog-agent/pull/18165/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R45, you might want to talk to @Gui774ume

Reply:
I'm in his team, so it's good to go like this. This commit is included in the new release